workitem/workitem_repository.go

```diff
@@ -538,7 +538,7 @@ func (r *GormWorkItemRepository) listItemsFromDB(ctx context.Context, criteria c
 		}
 		db = db.Limit(*limit)
 	}
-	db = db.Select("count(*) over () as cnt2 , *")
+	db = db.Select("count(*) over () as cnt2 , *").Order("execution_order")
```

Comment:
@DhritiShikhar So what did you check that made you conclude ASC is the correct order?